---
title: "Notes on coarse-graining the proteome of phage infected cells"
author: Griffin Chure
date: December 11, 2016
 output:
     pdf_document:
        latex_engine: xelatex
fontsize: 11pt
geometry: margin=0.75in
papersize: letter
---
# Coarse graining the proteome of phage infected cells

These notes correspond to my reading of [Hui et al. "Quantitative proteomic analysis reveals simple strategy of gobal resource alocation in bacteria"](https://www.ncbi.nlm.nih.gov/pubmed/25678603). I include my comments on the paper as a whole as well as at specific points of the research. I also include some of my musings about how this approach can be translated to understanding the dynamics and energetic cost of virus production.

## General idea outline
The authors address the idea of how expression of a 'useless protein' may impact how the proteome is broken into sectors of expression.  The entry of phage infected cells into the lytic phase  can be viewed as a case of extreme expression of a useless (actually damaging) protein. There is an enormous perturbation of the standard expression profile of cells. By examining how such a process may manipulate the expression profiles of cells, we could more accurately measure the energetic effects of viral infections. Furthermore, we may be able to extract some information regarding the burst size of infections.

## Specific comments from reading

### General approach
- The goal of the work was not to examine the relative abundance of specific enzymes and proteins under varying growth conditions, but to measure the fractional abundance of sectors of the proteome. One can think of each sector as a highly coarse grained enzyme that facilitates the turnover of material from one metabolic pool to the other.

- The authors state that the "key to this analysis is the concept of an 'effective concentration' for each coarse-grained enzyme, obtained as the fractional abundance of the sum of all its enzyme components among all expressed proteins in each condition." Mathematically, this translates to

  $$
  C_{eff} = \frac{\sum\limits_{i \in s}\text{enz}_i}{\sum\limits_{s \in p}\sum\limits_{i \in s}\text{enz}_{is}},
  $$

  where $s$ is the sector of interest, $i$ is an enzyme in that sector, $\text{enz}$ is the enzyme of interest, and $p$ is the total cell proteome. Rather than determining absolute protein copy numbers, they used coarse-grained spectral counts as a proxy. This procedure is described in [Malmstr√∂m et al., 2009](http://www.nature.com/nature/journal/v460/n7256/pdf/nature08184.pdf). This was an appropriate measure of abundance as the concentrations of the coarse-grained enzymes correlated linearly with the growth rate.

- The cells were grown in a variety of growth conditions in which bottlenecks were imposed on three of the four metabolic sections.
  - Catabolic sector -- growing on lactose and titrating LacY
  - Anabolic sector -- varying ammonia assimilation through titrating GOGAT.
  - Polymerization sector -- titrated sub-lethal amounts of translation inhibitor (chloramphenicol).
  - Central metabolism section -- no imposed bottleneck.

### Quantification methods
- The authors used ^15^N-labeling to provide relative quantitation of unlabeled proteins with respect to the labeled proteins. This means that for exach experimental run, equal amounts of the experimental sample was mixed with a known labeled standard sample as a reference. In this work, they verified the accuracy and precision of their work by using samples of unlabeled and ^15^N-labeled ribosomes.
  - The efficacy of quantifying whole-cel proteomes was verified by mixing ^15^N-labeled and unlabeled cells in fixed ratios. Again, this agreed very well  with the prediction.
  - They determined the effective precision of their quantification method to be $\pm$ 18\%.

- The method of spectral counting is relatively imprecise for quantifying single proteins, but is effective for groups of proteins (and therefore this analysis).
  - This method works by counting the number of peptides recorded for each individual protein as a measure of the absolute abundance.
  - Since their coarse-grained groups were massive, this error was miniscule. They state that "for a protein group comprising more than ~5\% of the proteome, spectral counting produces estimates with < 20% error."
  - They convince me that this method is sensitive enough by comparing their values for individual proteins with those estimated in [Gene Wei Li's 2014 paper on ribosome profiling](https://www.ncbi.nlm.nih.gov/pubmed/24766808).
  -

### Clustering results
- Instead of defining the clusters by hand, the authors defined clusters using the Pearson correlation as a distance metric. With this, they were able to cluster the proteome into five major clusters.
  - Most of the proteins appeared to respond specifically to a single bottle neck. The only exception to this was by one cluster (denoted by S) which was defined by protein levels that increased under both anabolic and catabolic growth limitations.
  - There was one group which did not appear to respond to any growth limitation, implying constitutive expression.

- The authors clame that the U-cluster shows to response to limitation. However, figure 2 shows that many of the entries in this cluster are actually have lower expression than the 'standard condition'. Is this a labeling issue?

- They examined what fraction of the proteome was occupied by each C$^\updownarrow$A$^\updownarrow$R$^\updownarrow$ group to refine their clustering.
  - They found three groups of responses which comprised a major fraction of the proteome asa function of growth rate. These were C$^\uparrow$A$^\downarrow$R$^\downarrow$, C$^\downarrow$A$^\uparrow$R$^\downarrow$, and C$^\downarrow$A$^\downarrow$R$^\uparrow$. The rest of the groups were minor constituents and below the 5\% proteome threshold for accuracy of the spectral method.
  - It is interesting that each of these major groups only respond to one type of bottleneck. Is there someway you can correlate this to infomration processing ability?

- To determine the biological function of each of these sectors, they used gene ontology analysis to assign functions to the abundant proteins in each sector. **Note that up to this point, there has been to real *knowledge* of what each protein does in a sector.** These sectors only exist due to clustering based off of their expression.
  - Note that each gene ontology term was weighted by that proteins abundance to make the roles of each sector more clear.
  - This abundance weighted approach means that 60\% of the mass fraction of each sector is accounted for by at most three gene ontology terms.

## Flux matching
- One of the author's primary arguments in the paper is that one can assign a flux of biomass through each sector allowing for coarse-graining of a sector as 'one' enzyme which feeds material to other metabolic processes.
  - They note that the growth rate dependence of each sector is very linear. Some sectors appear to all have the same-ish slope suggesting a more general brand of response. However, the three aforementioned 'significant' sectors have very specific responses when compared to the activity of the other sectors.
  - The fact that they are all very well described by a simple linear dependence on growth rate suggests a fundamental principle for the response to environmental challenges.

- Below, I go through the construction of the mathematical description of biomass flux. Using the idea of the six-component protetome,  we can state that the fractional abundance of a given sector $\phi_\sigma$ as a function of growth rate ($\lambda$) is
    $$
    \phi_\sigma(\lambda) = \phi_{\sigma, 0} + \Delta\phi_\sigma(\lambda),
    $$

  where $\phi_{\sigma,0}$ is the basal fractional abundance and $\Delta\phi_\sigma(\lambda)$ is the growth-rate dependent change in fractional abundance.  One can make the assumption that the flux of biomass $J$ through a sector $\sigma$ is proportional to the growth-rate dependent fractional abundance term $\Delta\phi_\sigma(\lambda)$,
  $$
  J_\sigma= k_\sigma\Delta\phi_\sigma,
  $$

  where $k_\sigma$ is a kinetic coefficient describing the efficiency of the sector. The authors argue that "the flux of a sector can be defined as teh sum of the metabolic products that flow out from the terminal enzymes per unit time, multiplied by a stoichiometric factor that reflects the composition of the material". This statement makes me feel a little bit odd because it is stating that the output from different sectors can be directly comparable which I don't believe is really true. For the R sector, the output is translated proteins while the output of the A sector is some set of amino acids. These two things procede at different rates and, more importantly, **have different energetic costs**. To continue the 'derivation' (for lack of a better word), we can examine the case of the R (protein synthesis) sector. Using what we've worked through above, we can write the flux $J_R$ through this sector as
  $$
  J_R = k_R\Delta\phi_R.
  $$

  In this case, $k_R$ is the kinetic parameter given by the peptide elongation rate in combination with the stoichiometric flux for cell growth. The latter can be written as
  $$
  c_RJ_R = \lambda
  $$

  where $c_R$ is the stoichiometric coefficient. Substituting this relationship into our definition of the metabolic flux, we arrive at
  $$
  \frac{\lambda}{c_R} = k_R\Delta\phi_R.
  $$

  Making some simplifications, we can formulate a description of the growth rate-dependent term of the fractional abundance to be
  $$
  \Delta\phi_R(\lambda) = \frac{\lambda}{\nu_R},
  $$

  where $\nu_R = c_Rk_R$ is an effective rate constant for the R sector. This can be applied to the U sector as well as there is no specific response to the applied bottlenecks. By this definition, the general response for the R sector under the catabolic and anabolic growth limitations should be identical, which matches well with their data.

  The case for the C, A, and S sectors are more complicated as they not independent. That is to say that the S sector (C$^\uparrow$A$^\uparrow$R$^\downarrow$) contains proteins which respond to both catabolic and anabolic growth limitations. This can be accounted for in the flux model by considering a set of proteins which respond to the catabolic limitation $\mathbf{c}$ and a set of proteins which respond to the anabolic limitation $\mathbf{a}$. By pumping these into the same linear relationship between flux and growth rate, we arrive at
  $$
  \begin{align}
  \Delta\phi_\mathbf{c}(\lambda) &= \phi_{\mathbf{c},0} + \frac{\lambda}{\nu_C}\\
  \Delta\phi_\mathbf{a}(\lambda) &= \phi_{\mathbf{a},0} + \frac{\lambda}{\nu_A}
  \end{align}
  $$

  The components of the S sector include those from C and A. However, note that those protein components of the $\mathbf{c}$ and $\mathbf{a}$ sets **only** respond to catabolic and anabolic growth-rate limitation respectively. With this conservation-of-mass in mind, we can write the growth rate-dependent fractional abundance terms for the A, C, and S sectors as
  $$
  \begin{align}
  \Delta\phi_C(\lambda) &= (1 - f)\frac{\lambda}{\nu_C},\\
  \Delta\phi_A(\lambda) &= (1 - f)\frac{\lambda}{\nu_A},\\
  \Delta\phi_S(\lambda) &= f\left(\frac{\lambda}{\nu_C} + \frac{\lambda}{\nu_A}\right),
  \end{align}
  $$

  where $f$ is the fraction of $\mathbf{c}$ and $\mathbf{a}$ sector proteins in common with the S sector. These equations therefore describe the general response of the sector flux to growth-rate limitations provided that one growth-rate limitation affects only one $\nu_\sigma$. To complete the model, the authors address the sector which shows no growth rate dependence as
  $$
  \Delta\phi_O(\lambda) = 0.
  $$

 - A result of this model is that it describes the specific responses of the sectors to the corresponding growth-rate limitations as well as the general response, without introducing any other parameters, without introducing any other parameter, without introducing any other parameters. An important point to note, however, is that the fraction of the proteome that is available to any one sector is limited by the fact that
  $$
  \sum\limits_\sigma\phi_\sigma = 1.
  $$

  This means that the maximum change in the growth rate-dependent abundance fraction is determined by the basal abundances of each fraction. Mathematically, this translates to
  $$
  \sum\limits_\sigma \Delta\phi_\sigma = 1 - \sum\limits_\sigma\phi_{\sigma,0} \equiv \phi_\text{max}.
  $$


##  Testing under different growth conditions.

- In typical Terry Hwa fashion, the authors posit that some of their model parameters ($\nu_\sigma$s) are dependent on the growth medium while some components ($f$ and $\phi_{\sigma,0}$) are independent of medium. To test this hypothesis, the authors redid the catabolic and anabolic growth-rate limitations using glycerol medium as the standard condition. If their model were correct, they would expect that this perturbation would only affect $\nu_C$ with respect to the glucose standard condition.

## Effect of protein overexpression
- One would expect that the fractional abundance of each sector is very tightly connected to everything that is being expressed in the cell. Therefore, if one were to overexpress a useless protein (in the paper they used LacZ), the only parameter that should be affected is $\phi_\text{max}$ since the only thing that is changing si what the cap on abundance can be.

- Their data from this experimental modification is a close agreement between prediction and experiment.

- This is a pretty minor case of overexpression. What about when it's far more severe? Does it just halt all other protein expression?



## Application to phage infection
Below are some questions about how this approach can be applied to studying some of the proteome dynamics of phage infection.

- Do the timescales make sense for performing this type of analysis? If the exit from lysogeny is very fast (which I believe it is), one may not see any effect on the proteome of the cell. Rather, you'll just see a massive uptick in expression of other things while the relative proportions of the cell proteome are the same.
  - Furthermore, this kind of experiment only works when you have control over the growth rate (at least in the way that they did it). However, in the case of phage infection, their "extreme limitation" case is met with $\lambda \rightarrow 0$ (actually, I think it is $\lambda \rightarrow -1$).


- Would it be possible to examine the dynamics of expression? I believe one can make synchronous lytic cultures of $\lambda$ infected bacteriophage. Would it be possible to induce entry into the lytic phase and then take 'snapshots' of expression by hitting the cultures with translation inhibitors? One could then run each snapshot through quantitative mass spectrometry to estimate the abundance of each component of the phage $\lambda$ proteome.


## What to read next?
- The authors continuously reference [this paper](http://www.nature.com/nature/journal/v500/n7462/pdf/nature12446.pdf) (also from Terry Hwa's lab) which took a somewhat similar approach to dissect the physiological consequences of cAMP signalling in *E. coli*.
