\documentclass[letterpaper]{article}
\usepackage[margin=0.75in]{geometry}
\usepackage{graphicx}
\begin{document}
One of the most fundamental aspects of biological inquiry is the relationship
between the sequence of the genome and the physiology of the organism. While we
are still trying to understand the sequence-structure-function relationship of
the genome, there is still much to be understood about how genomes are built and
organized. Whether by viral infection, conjugative transfer, or direct uptake of DNA
through transformation (Fig 1 A), bacteria frequently share their genetic material with
their neighbors (including those of different species) influencing the
architecture of the recipients' genome. How this horizontal transfer
of genetic information has influenced their evolution, however, remains enigmatic.

Using some
clever tricks of molecular biology, we have engineered a system that allows for
sequence specific visualization of DNA molecules \textit{in vivo}. Our system
exploits the ParBS plasmid partitioning system to allow for the subcellular
localization and detection of plasmids in living cells without relying on a
secondary reporter associated with the plasmid of interest (Fig 1 B).
Fluorescently labeled ParB molecules are constitutively expressed in the cells
of interest. Upon the uptake of DNA containing a \textit{parS} sequence, ParB
specifically binds and quickly oligomerizes allowing for a large number of ParB
molecules to be concentrated over a short stretch of DNA forming a fluorescent
puncta within the cell. The association
between ParB and \textit{parS} is rapid  and allows for instantaneous
measurement of the uptake of DNA.

This system has been successfully employed in \textit{E. coli} (Fig 1 C) and studies
are currently underway to study replication and segregation of plasmids in
dividing cells as well as the frequency of phage transduction and it's
dependence on location in the genome. We are in the process of integrating the
\textit{parS} sequence into various locations of the \textit{E. coli} genome to
observe the relationship between genomic position and transduction by P1 phage.


We are also in the
process of translating this system into \textit{Bacillus subtilis} whose
competence state is well characterized and understood. Despite decades of study, how many
competent cells actually transform and how many DNA molecules taken up in a
transformation is still unknown. Once this system is established in
\textit{Bacillus},
we can very easily make measurements of how frequently DNA is taken up. This can
be done to either measure the efficiency of uptake by doing a transformation in
bulk and then observing transformants using microscopy or by watching the uptake
occur in real time by growing the cells under the microscope in an environment
containing \textit{parS}-plasmids.

\begin{figure}[!h]
	\centerline{
		\includegraphics[width=0.75\textwidth]{HGT_fig.png}}
	\caption{{\small Single-cell studies of HGT in bacteria. Three main
		classes of HGT in bacteria are shown in panel \textbf{A}. While
		conjugation has been studied extensively, much less is known
		regarding the occurrence and mechanism of transformation and
		transduction. The event of recombination into the host genome is
		critical or persistence as a plasmid is critical to the
		evolutionary importance of the gene transfer.\textbf{B}.
		The uptake and subsequent partitioning of foreign DNA elements
		into daughter cells. Question marks indicate processes to be
		studied in this research. Panel \textbf{C} shows
		fluorescent-ParB expressing \textit{E. coli} containing a
		\textit{parS} plasmid growing on an agar pad.  Bright puncta
		represent the location of the plasmids within the cell.}}
\end{figure}


\end{document}
