**Single Molecule Investigation of Phage-Mediated Gene Transfer in *E.
coli***

Understanding the transfer of heritable information from parent to
offspring has long been the focal point of biological research. Vertical
gene transfer, the transmission of genetic determinants from ~~one
generation to the next both through sexual and asexual reproduction,~~
parent to offspring ~~has long been the focal point in the field of
genetics.~~ has served as the “standar model” of evolution. Our
understanding of gene transfer, especially in eukaryotes, is mostly
built on this model. For quite some time, ~~As a result~~, it was
assumed that all life evolved in this manner, including prokaryotes.
~~prokaryotic gene transfer~~ ~~is fundamentally vertical as well~~.
However, in 1928 Frederick Griffith showed that pathogenicity could be
transferred *horizontally* between populations of bacteria, suggesting
that vertical inheritance was not the complete story. This work was
expanded upon by Oswald Avery and Colin MacLeod who showed that DNA was
the chemical identity of Griffiths’ “transforming material”. With the
advent of reliable genome sequencing, it has become incredibly apparent
that genes have been transferred between species, phyla, and even
domains over the course of evolutionary history. I has been estimated
that in the course of proteobacterial evolution, 75% of all protein
families have experienced 1.9 horizontal gene transfer events (citation
in my NSF proposal). Vertical gene transfer, and its implications with
respect to evolution has been studied extensively and is well
understood. Over the past several decades, horizontal gene transfer in
prokaryotes has received a considerable amount of study, yet there is
still much that remains enigmatic. Many interesting and basic questions
surrounding HGT can be answered by observing the events at the
appropriate resolution. While population scale studies are informative,
being able to accurately describe the movement of DNA through HGT at the
cellular level will not only further our understanding of the origin and
passage of genetic information, but of bacterial evolution as a whole.
~~In contrast, the mechanisms of horizontal gene transfer are
considerably more enigmatic. have been not been studied as extensively.
~~

~~Lateral gene transfer plays a major role in the evolution of
antibiotic-resistant bacteria, making the study of lateral gene transfer
pertinent to health-care discussions today. However, prokaryotes exhibit
other specialized processes for gene transfer, for example natural
transformation, conjugation, and transduction – these processes are
termed lateral gene transfer, or horizontal gene transfer. In general,
lateral gene transfer refers to the transfer of genes from one organism
to another organism which is not an offspring of the first organism~~.

**This portion can be split up into three paragraphs: Natural
transformation (uptake of naked DNA – midrange transfer), Conjugation
(DNA transfer through a pilus – short range transfer), and Transduction
(phage mediated transfer – long range). The final paragraph should be a
bit longer than the other two since it is your subject. I’ve kind of
broken it up for you below and added a few bullet point of what you can
discuss. You’ll have to do some research on this yourself but I’m always
available if you need some guidance. **

Natural transformation occurs when a cell picks up exogenous DNA from
its environment and integrates the DNA with its own genome. This process
is rife with challenges as the incredibly hydrophilic DNA needs to cross
the remarkably hydrophobic barrier of the cell wall, evade the
restriction systems of the host bacterium, and either integrate into the
chromosome or circularize and persist as a plasmid. ~~The chemical
structure of DNA makes it extremely hydrophilic, while the chemical
structure of cell membrane makes it extremely hydrophobic~~. As a
result, the cell must be in a specific state, called the “competence
state”, in order to be able to experience transformation. **Perhaps you
could spend another sentence or two explaining that the competence state
requires an incredible amount of communication between genetic circuits
to build the appropriate machinery to facilitate DNA transfer. I would
also add that the definition of a “competence state” is very vague. Many
organisms genomes show that horizontal gene transferred yet they lack
the cellular machinery characteristic of the competence state in other
organisms. **

Another method of horizontal gene transfer is bacterial conjugation, in
which a plasmid is transferred from one cell to another through a
tube-like bridge or direct contact between the cells.

**Certainly expand on this section by a few sentences. The Wikipedia
page for conjugation gives a pretty good review. Maybe that could guide
you.**

In both natural transformation and conjugation, the cell must be close
to the source of the donated genetic information, be it the environment
or some other cell. Transduction is the only known method of long-range
gene transfer, **I would describe what you mean by ‘long range’** and is
the topic of my research. Transduction is the process in which genetic
material is transferred from one bacterium to another through a virus
proxy, called a bacteriophage. Once the virus infects the cell,
transduction can then enter one of two states, the lysogenic state or
the lytic state. In the lysogenic state, the bacterium integrates the
foreign genetic material into its own genome, where it can sit for
generations while the cell continues to live and reproduce normally. In
the lytic state, the viral DNA is not integrated into the cell’s genome,
but continues to replicate until the cells bursts and releases the phage
into the environment, where it can infect other bacteria. Transduction
is believed to be a significant factor in the development of mosaic
genomes, making it a process of interest in the field of genetics. While
there have been studies on the injection of DNA into a cell,
transduction at a single cell resolution has yet to be studied in depth.
Furthermore, the frequency at which genes are transferred between
bacteria has not yet been determined. **You should mention specifically
that viruses sometimes accidentally package segments of the HOST genome
into it’s capsid. When this virus infects another cell, it cannot
replicate and can only donate the segment of the host genome to the new
bacterium. Viruses can only enter the lytic or lysogenic state if they
contain their own genome by definition. I think this point should be a
little more clear.**

Transduction has been known of for decades and our current understanding
comes bulk-scale measurements. The frequency of transduction reported in
these types of experiments are almost certainly underestimates of the
actual frequency of transduction. My project seeks to study transduction
on a single-cell level by watching fluorescently labeled segments of DNA
be transferred from virus to cell, integrate into the host genome, and
shape the genetic diversity of the subsequent bacterial generations.
~~in depth~~. ~~Firstly~~, (**I’m not a big fan of that word)** I would
like to measure the frequency at which bacterial genes are transferred
by the process of transduction. Then, I would like to determine if there
differences in gene transfer frequency depending gene type – core genes
or accessory genes. Once the frequency of transduction is determined, I
would like to identify how different conditions affect transduction.
Specifically, how does selective pressure affect transduction? Do
stressful states lead to a higher frequency of successful transfer? Does
the presence of an antibiotic in the environment affect the transduction
in any way?

The frequency that transduction occurs has been estimated at a bulk
scale. However, it has yet to be studied at single-cell resolution. In
order to study transduction at this scale, we will need to use a phage
that will always enter the lytic state so that we may determine if a
bacterium has been infected – the bacterium will surely burst and die if
that phage is in the lytic state. The differences in gene transfer as a
function of gene type can be measured both in bulk and at single-cell
resolution. To perform this measurement, we will need to insert a
reporter system at several sites on the genome and measuring the
frequency that it is found. Studying the impact of different
environments can both be done at the single-cell and bulk level.

I also wish to study the dynamics of recombination into the host genome.
Being able to visualize recombination at the single-molecule level would
provide valuable insight on the frequency of successful integration.
This requires a system that is sensitive to the movement of DNA
throughout the cell. Using the system described below, we will be able
to resolve free DNA vs recombined DNA using the difference in diffusion
between them. **This paragraph is a little weak, but you get the idea
.**

![](media/image1.png)**The approach section needs to be expanded a
little bit. A figure would certainly help here, which I have attached in
the email and have included here. **

I plan on using ~~conventional~~ single-molecule/single cell microscopy
techniques to observe the transfer of specific DNA regions into other
cells through the action of transduction. The lab where I plan on
working currently has a system in which a protein, *parB*, has been
fluorescently tagged with YGFP that binds specifically and cooperatively
to a parS, a DNA sequence. Upon binding of *parB* to the *parS* site,
*parB* quickly oligomerizes resulting in a high concentration of protein
in a relatively small volume. Under the microscope, these appear as
bright concentrated spots of fluorescence, in contrast to the uniform
fluorescence that is exhibited when the DNA sequence is not present.

For the phage experiments, we will need to integrate this *parS* site to
many different places on the chromosome. Recombineering in *E. coli* has
been incredibly well engineered and similar methods should be able to be
applied to the integration of this *parS* site. This system will be able
to serve as an instantaneous reporter. Typically, we would only be able
to determine the expression of our reporter gene after the maturation
time of the reporter. However, because the *parB* protein is continually
expressed, we can instantaneously determine whether *parS*-DNA is
present. The *parS* sequence is very small meaning that the genetic
region we are observing is relatively unperturbed. Studies have been
done previously to monitor plasmids by having an array of \~256 *lacO*
or *tetO* binding sites to which reporters attach. This results in about
12kb of excess DNA. In our case, we are really only modifying the
genomic environment by \~100bp.

The bulk assays will be done using traditional biological methods. This
can be done by plating onto selective plates and counting colonies as a
function of the total number of surviving cells.

> Image processing and data collection will be written in Python.

After we finish collecting data, we will be able to build a quantitative
statistical model describing the frequency at which genetic information
is transferred through transduction.

![](media/image2.png)**The schedule section could also be simplified
with a figure (also inserted). **

The unique construction of this research allows for multiple objectives
to be studied at once. For the first week and a half of my research, I
will becom familiarized with the microscopy equipment in the lab along
with the specifics of molecular biology and raising of bacteriophage
stocks. The DNA visualization system described in Figure 1A has already
been constructed allowing for the beginning of experiments immediately.
For the entirety of the research, I will clone the *parS* DNA sequence
into various regions of the *E. coli* host genome. **Blah blah you get
the idea here. Spend a few more sentences explaining that measurement of
the dynamics of recombination can be performed while making measurements
of the frequency of transduction. **

I plan on using the first two weeks to familiarize myself with
microscopy, generation of phage stocks, and general microbiology
techniques. I will also integrate *parS* sequences into accessory loci.
In the next two weeks, I will conduct both bulk experiments and
single-cell experiments, in which I will infect bacteria with phage
carrying bacterial genome and continue to integrate *parS* sequences
into accessory loci.
