%%%%%%%%%%%%%EXTRA PREAMBLE SHIT%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\input{../../tex_tools/candidacy_preamble.tex}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%TITLE STUFF%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title{The Mobile Genome: Probing Horizontal Gene Transfer at Single-Cell
Resolution}
\author{Griffin Chure - Research Candidacy Proposal\\
	\textit{\small California Institute of Technology - Biochemistry and Molecular
	Biophysics}}
\date{\small Draft -- \today}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\maketitle
\begin{abstract}
	Bacterial populations live in a fertile soup of foreign genetic
	material. Whether by viral infection, conjugative transfer, or
	direct uptake of DNA through transformation, bacteria frequently share
	their genetic information with their neighbors, including those of
	different species, strongly influencing bacterial evolution. In this
	research, I seek to characterize and quantify the frequency and dynamics
	of the horizontal transfer of genetic material at single-cell
	resolution. I have employed a genetically encoded system in \textit{E.
	coli} and \textit{Bacillus subtilis} that allows for the observation of
	specific DNA molecules as they are transfered between individual cells.
	The orthogonal nature of the visualization system allows for \textit{in
	vivo} studies of conjugation, transformation, and transduction. In
	addition, this research will advance our understanding of the
	evolutionary consequences horizontal gene transfer on the formation of
	microbial communities. 
\end{abstract}

\section*{Introduction}
Understanding the ebb and flow of genetic information through lineages and
communities has long been a major focal point of biological inquiry.  
The genome, however, is not limited only to travel through a family tree.
Horizontal Gene Transfer (HGT) is the transmission of genetic material, whether
it be informational or operational, from one organism to another through means
other than sexual or asexual reproduction. In essence, it is the assimilation 
of foreign genetic information into another organisms genotype, for better or
for worse. HGT allows for an immediate variability in genotype not possible only
through mutation and descent with modification. Antibiotic resistance cassettes
are the most cited examples of HGT, although the range of genetic information
that can (and has) been transferred is vast and it's role evolution is still not
well understood.

While HGT was first discovered by Frederick Griffith in 1928 as he studied the
transformation of non-pathogenic \textit{Pneumococcus} into is nefarious
counterpart\cite{Griffith:1928vg}, the extent of HGT was not understood until
the first genomes were completely sequenced. The presence of "genomic islands"
-- regions of the genome with unusual G+C content or aberrant codon usage --
were surprisingly abundant and often encoded functional gene products such as
secretion systems and pathogenicity cassettes (so frequent they garnered their
own terminology of pathogenicity islands\cite{Hacker:1990}). While HGT occurs at
a higher frequency and has received extensive study in prokaryotes, the
eukaryotes are not immune to this powerful force of evolution. The most obvious
example of HGT within the eukarya is the genome reduction and gene shuttling
from endosymbiont to host which occurs at surprisingly high frequency (reviewed
in \cite{Timmis:2004ks}).  Prokaryote-eukaryote HGT events (apart from bacterial
endosymbionts) are common and the number of genes acquired in this manner can
range from zero to hundreds depending on the eukaryotic
species\cite{Keeling:2008ch}. Likewise, HGT from eukaryote to prokaryote has
also been reported\cite{Doolittle:1990uo} the most astonishing of which is the
encoding of alpha- and beta-tubilins along with the kinesin light chain in the
bacterium \textit{Prosthecobacter}\cite{Jenkins:2002ty}. Furthermore there have
been several reports of HGT within the eukaryotes (reviewed in
\citet{Keeling:2008ch}) although most events appear to  replace genes with
existing homologs rather than confer new functions. Our current understanding of
eukaryote-eukaryote HGT is almost certainly an underestimate but will continue
to improve as more genomes are sequenced and published.

Prokaryotes have been the classical choice for probing HGT for good reason.
The short generation times, physical size, and modern molecular-biological tool
set make bacteria the model organisms for studying not only the molecular
details of how the DNA is transported into the cell but for a quantitative
dissection of the fitness effects such transfers can cause. While bioinformatic
approaches have greatly helped us understand what has been transferred, it is
often difficult to infer how the transfer occurred. More importantly, the
information produced by these studies reveal only the horizontal transfer events
which did not confer a deleterious change in fitness to the host. It is likely
that HGT is a far more common occurrence than the genomic sequence would suggest
as we frequently only observe the events which "stuck".

Understanding the dynamics and extent of HGT in microbes is not only important
for our understanding of evolution and microbiology as a whole, but is vital to
combating the spread of multiple-antibiotic resistant pathogens. In 2014, the
World Health Organization declared the global rise in antibiotic resistance for
as the most pressing issue in human health in the past XXXX.The following
sections will describe the primary mechanisms of HGT in the microbial world
(Fig. \ref{fig:mechanisms}),
special cases which have only recently been discovered, and the evolutionary
implications HGT which tangle the branches of the microbial "tree-of-life".


\subsection*{Conjugation}

Through the early 1940's, the understanding of bacterial phylogeny and
prokaryotic inheritance as a whole had been limited to the accumulation of
mutations through vertical transmission of genetic information from cell to
cell. The first evidence of genetic recombination in bacterial specimens came
from the work of Tatum and Lederberg in 1947 which showed that bacteria can
transfer information with their neighbors by entering a "sexual
stage"\cite{Tatum:1947va}.In the 70 years since the discovery, the molecular
details of bacterial conjugation have been studied extensively, although the
evolutionary implications of this method of transfer remains unclear. 

Conjugation is the highly-specific transfer of specific DNA elements from a
donor to acceptor cell through a fragile membrane connection 


Because a F element is needed with a specific start and end site, cells are
limited in what genes can be transfered. Unlike in transformation and
transduction, it is very specific in it's transfer. 

Fertility element (HFR) encodes all genes necessary for transfer.  This is also
the genetic element that gets transfered. This means that so long as it
successfully transfers in it entirety, the receiving cell will become able to
pass along the plasmid. 

The plasmid has a specific site \textit{oriT} where the transfer and begins and
ends. It is at this site that relaxase proteins nick the plasmid allowing a
single strand to be passed through the pilus.  HFR-HFR transfers are rare or do
not occur at all (NEED CITATION).  This indicates that there is some molecular
marker which HFR cells can identify viable targets.  Primary mechanism for
horizontal gene transfer. This is very short-range, however, as the cells need
to be close enough to touch each other. This means that conjugal transfer in
biofilms is expected and incredibly common.  Capable of transferring an entire
genome into a recipient. This is dependent on the stability of the cell
suspension as the pilus connection is fragile and susceptible
to mechanical stress.

For mobile elements to be self-transmissible, they must contain information for
maintenance, dissemination, and regulation \cite{Burrus:2004ca}. Unlike
plasmids, integration into the chromosome of it's host ensures vertical
transmission into the hosts' progeny. The probability of loss is much lower than
in the case of plasmids which are autonomous elements in most cases. Most ICEs
contain a recombinase which integrates into a specific locus on the host genome
(\textit{attB}) and a locus on the ICE (\textit{attP})\cite{Burrus:2004ca}.


What are the open questions?

%\begin{itemize}
%	
%	\item ICEs are mobile elements which have been integrated into the
%		chromosome and have the ability to be transfered via
%		conjugation, but do not necessarily encode the conjugation
%		machinery. 
%	\item ICE frequently contain another factor, excisionase (\textit{Xis}) which is
%		responsible for the excision of the ICE from the host chromosome
%		after integration. WHAT IS THE FUNCTION EXACTLY?
%	
%	
%%DNA is most frequently transferred between cells during
%%conjugation as a single strand. This requires four components -
%%the relaxase for nicking and binding of the ssDNA, a coupling
%%protein which binds and transfers the ssDNA between the donor
%%membranes, and a type IV seqcrion system (T4SS) which establish
%%the contact between the two cells and serves as a conduit for
%%the ssDNA to transfer (CHECK) \cite{Guglielmini:2014bc}. As T4SS
%%are highly diverse structures and are widely distributed across
%%bacterial species, not all mobile elements encode T4SS and use
%%them \textit{in trans}.
%
%\end{itemize}

Many of the experiments which characterized this system were done either \textit{in
vitro} or in bulk, leaving many questions regarding the cellular and molecular
events unanswered. Such questions, such as the frequency of conjugation in a
mixed population of donors and recipients still remain unanswered.  

There have been some \textit{in vivo} which have helped shed light on how the
conjugal elements traverse bacterial communities. Babi\'{c} et al., 2008 used
real-time fluorescence microscopy to measure the frequency of conjugative
transfer within a mixed population of donors and acceptors. To visualize the
horizontally transferred DNA, the authors created a SeqA-YFP fusion which
specifically and strongly binds hemimethylated DNA. DNA from an F+ Dam+
\textit{E. coli} strain was mixed with an F- Dam- recipient. The
methylated DNA transferred into the recipient was bound by SeqA-YFP upon
formation of the hemimethylated duplex forming a bright foci within the cell.
This allowed for observation of transfer of any DNA sequence. While this proved
useful for measuring the frequency of transfer, it provided little-to-no
information regarding what was transferred. The authors observed transfer
happening over a very short period of time after incubation and up to great
distances apart (with a maximum of 12$\mu$m), proving that DNA can be
transferred through the F pilus directly.

This shows that single-molecule and single-cell approaches to studying
horizontal gene transfer can provide deep insight into both the molecular
mechanisms as well as the frequency of transfer \cite{Babic:2008bl}.


\subsection*{Transduction}
Bacteriophages are the most numerous forms of life on the planet. The global
population is estimated to be on the order of 10$^{31}$ phage
particles \cite{Comeau:2008jo}. This population is estimated to perform 2$\times
10^{16}$ transduction events per second \cite{ChibaniChennoufi:2004gv}.


Sequence based methods are often used to infer phage-mediated gene transfer.
Genomic islands in an array of marine cyanobacteria are flanked by viral-like
sequences, suggesting they were transferred through a transduction event
relatively recently in their evolution\cite{Anonymous:2004vg, Palenik:2003ef}.

The horizontal transfered can also have a profound effect on the evolution of
viral genomes. While initially considered an oddity and attributed to
contamination of free bacterial DNA, metagenomic and genotyping studies showed
that a myriad of cellular genes have been incorporated into bacteriophage
genomes such as the photosystem II core reaction center and electron transfer
genes\cite{Anonymous:2004vg,Comeau:2008jo}.  This abundance of cellular genes
buried within the phage genomes emphasizes the power phage can have in
influencing the metabolic repertoire of their host. 

The benefit is not only felt by the host bacterium, however. A body of evidence
is being generated showing that transcription of cellular genes carried by a
phage can be transcribed immediately after infection. 

Compare and contrast generalized and specialized transduction.

\begin{figure}
\centerline{\includegraphics[width=\textwidth]{figs/mechanisms.eps}}
\caption{The primary mechanisms of HGT. \textbf{Conjugation},
\textbf{transduction}, and \textbf{transformation} are illustrated in the above
panels from left to right respectively. While the molecular machinery for each
mechanism of transfer is vastly different, each is capable of drastically
changing the genomic landscape. A symbol legend is present at the bottom of each panel.}

\label{fig:mechanisms}
\end{figure}
\subsection*{Transformation}
Being one of the largest and most hydrophilic biomolecules in nature, the
transfer of DNA from the environment into the crowded environment of the
cytoplasm is no easy task. After traversing the rigid cell wall and the
hydrophobic barrier of the cell membrane, the DNA must evade the myriad of
restriction systems evolved to degrade the potentially hazardous foreign
material and either circularize as a plasmid or integrate into the host
chromosome. In spite of these challenges, the uptake of free DNA from the
environment into the cytosol (known as genetic transformation) is a common
occurrence in natural environments. Transformation was first observed by
Frederick Griffith in 1928\cite{Griffith:1928vg} upon noticing that incubation
of heat-killed pathogenic \textit{Pneumococcus} with a living yet non-pathogenic
variant yielded pathogenic organisms. While  Griffith himself was not able to
identify the chemical identity of the "transforming principle", Oswald Avery,
Colin MacLeod, and Maclyn McCarty were able to prove that this substance was DNA
(not RNA or protein)\cite{Avery:2014wx} leading to the conclusion that DNA was
the hereditary material of life. Unfortunately, their results were initially
disregarded as protein as hereditary material was the popular opinion until the
experiments of Alfred Hershey and Martha Chase nearly a decade later
\cite{Hershey:1952}. In the nearly 100 years since it's discovery, the process
of DNA transformation remains a topic of extensive research at the biochemical,
systematic, and evolutionary level.


Transformation can occur via two fundamentally different processes --
artificially via chemical and physical shock or naturally through entry into a
genetically regulated physiological state known as the competence state. While
artificial transformation is used ubiquitously in a molecular biology context,
chemical and electrotransformation has been shown to be a viable route for HGT
in natural environments. Chemical transformation requires both a chemical and
temperature shock to cells, resulting in permeabilization of the plasma
membrane, allowing the diffusion of plasmids and linear DNA fragments into the
cytoplasm (FIND A CITATION). Despite its ubiquity, the precise mechanism
chemical transformation is still not well understood, yet this phenomenon has been 
proven to be incredibly valuable in  biological research.  A standard protocol
requires the somewhat harsh treatment of cells by washing and incubating cells
in non-physiological CaCl$_2$ concentrations along with large temperature shocks
(often $>$ 40$^\circ$ C). This protocol has been optimized for laboratory
purposes and one would be hard pressed to find these conditions in natural
environments.  However, chemical transformation can occur at lower
concentrations of CaCl$_2$ and smaller temperature shocks, both of which are
present in natural environments. \citet{Baur:2006ba} showed that \textit{E.
coli} cells can readily be transformed in natural freshwater with low
($\sim$2mM) CaCl$_2$ concentrations and temperature fluctuations (0 - 10$^\circ$
C) that are typically observed on a daily basis. This work was expanded upon by
\citet{Woegerbauer:2002ev} who showed that clinical isolates of \textit{E. coli}
could be transformed in mineral water, but not in clinically relevant media such
as urine. The extent of chemical transformation in nature is undoubtedly
sensitive to the geochemistry of the ecosystem, but has the potential to serve
as another avenue of HGT.

Electrotransformation is another method of artificial transformation in which
an  electric field (often $\sim$ 1800 V/cm) is applied to a DNA
and cell mixture suspended in a low ionic strength medium. The application of
pulsed electric fields results in the transient permeablization of the membrane,
allowing the diffusion of DNA and other extracellular components into the
cell\cite{Wegner:2015fx}. Much as in chemical
transformation, the preparation of electrocompetent cells requires
non-physiological treatment including successive washes with deioninzed water to
remove ions that would lead to electrical discharge. While it may be hard to
imagine an environment that would be conducive to electrotransformation, it has
been shown that such conditions can be found in soil microcosms through
lighting-mediated current injection\cite{Demaneche:2001kb}. Given that
global soil lighting strikes occur approximately 44 times per
second\cite{NOAAlighting}, it is not unreasonable to believe that
electrotransformation is a naturally occurring phenomenon. So-called
"Lightning-competent" \textit{Pseudomonas} strains have been isolated from
natural soil environments \cite{Ceremonie:2004gr} suggesting that natural
electrotransformation is more than conjecture.


Unlike artificial transformation, Natural transformation through genetic
competence is very well understood in several model organisms and it's relevancy
in natural environments is indisputable. Competence is found across many phyla
of bacteria with Gram-positive and Gram-negative members equally represented. As
of this writing, there are approximately 80 species known to exhibit
competence\cite{Johnston:2014dc}, although the evidence in some cases is a
single report.  The uptake machinery is best understood in \textit{Bacillus
subtilis} and \textit{Neisseria gonorrhea} and serve as the archetypal systems
for Gram-positive and Gram-negatives respectively. Regardless of the profound
differences in the cell envelope composition, both Gram-negatives and
Gram-positives share similar mechanisms of DNA transport and processing. Due to
the presence of a double membrane, transformation occurs in two steps in
Gram-negative bacteria -- DNA uptake (passage of dsDNA across the outer
membrane) and DNA translocation (passage of ssDNA across the inner membrane). In
both Gram-positive and Gram-negative bacteria, free DNA in the environment is
bound to the exterior of the cell through interaction with surface fibers of the
Type IV pilus family (T4P)\cite{Burton:2010eea}. T4P complexes are used by many
bacteria in "twitching motility" -- a flagella-independent method of movement
across moist surfaces\cite{Mattick:2002il}. A competence-specific pseudopilus
found in both Gram-negative and Gram-positive bacteria has been reported,
although it's role in free DNA recognition remains
unclear\cite{Anonymous:_TSDp-Ca}. After being bound to the cell surface, the
dsDNA is pulled into the periplasmic space in Gram-negatives (or to the inner
membrane in Gram-positives) where it is bound by a channel within the inner
membrane. One strand of the dsDNA is hydrolyzed as the other strand is brought
into the cytoplasm where it is either degraded by restriction systems,
circularized and replicated into a plasmid, or integrated into the chromosome
via homologous or non-homologous recombination.


In general, DNA uptake is non-specific allowing for inter-species, inter-domain,
and even inter-kingdom transformation. However, two natural competence systems
appear to preferentially take up DNA from related organisms by recognizing a DNA
Uptake Sequence (DUS) (reviewed in \citet{Mell:2014dj}). This phenomenon is
found in the \textit{Neisseria} and \textit{Pasteurellacae} families (both of
which are Gram-negative) who recognize short 9 - 10 bp motifs in exogenous DNA.
Upon sequencing of these genomes, it was found that these same motifs were found
scattered throughout the genome with a frequency of aobout one occurrence per
kb.  \textit{Neisseria} and \textit{Pasteurellacae} preferentially uptake DNA
from related genera. This is achieved by recognizing a specific uptake sequence
that is similar to repeats present in the respective genomes of each. How this
sequence is recognized still remains unknown. A single sequence is sufficient to
allow the transport of both short and long DNA fragments, suggesting that DUS
recognition is important only for initiation of DNA transport. Of the DUSs found
within the genome, one-third are within open reading frames and are trascribed
and translated into specific tripeptides. The proteomes of \textit{Neisseria}
and \textit{Pasteurellacae} show a distinct enrichment of these peptides
compared to other organisms which lack a DUS (such as \textit{E. coli}),
illustrating that uptake specificity can exert a powerful force on genome
evolution. There are numerous reasons in which uptake specificity may have
occurred. It is important to note that specificity in uptake is found in only
these organisms. Interestingly, the \textit{Neisseria} and
\textit{Pasteurellacae} are distantly related. Their DUS differe significantly
in composition, suggesting that specificity has evolved independently.


Entry into the competence state is a metabolically expensive endeavor which begs
the question of why it evolved in the first place. While we still do not 
It is possible that competence and natural transformation evolved to support
genetic diversity\cite{Barton:1998uq, Otto:2006vm}, to allow the use of DNA as a
carbon source\cite{Dubnau:199vq, Redfield:2001vx}, or to use free DNA as a
template for repair\cite{Claverys:2006do, Dorer:2010tf}.
\subsection*{Special Cases}
While conjugation, transformation, and transduction are the primary ways which
DNA is horizontally transfered, there are other mechanisms which appear to
facilitate HGT, although the precise mechanism and the extent at which they
occur is unknown.

 Gene Transfer Agents (GTAs) are  phage-like particles that package random dsDNA
 fragments from the chromosome into a capsid-like structure which are released
 into the environment through cell lysis (reviewed in \citet{Lang:2012df}). While
 reminiscent of generalized phage transduction, the production of these
 particles is not the product of a viral infection as all components needed for
 their assembly is encoded within the producing cells' genome. Additionally, the
 amount of DNA that can be packaged is less than the length of the GTA encoding
 genes, meaning that the particles cannot be self perpetuating (although the GTA
 operon maybe reconstituted upon multiple infection, albeit unlikely). Four
 naturally occuring GTAs have been discovered and all appear to be unrelated
 evolutionarily. Their structural and functional relationship with known
 bacteriophages is indisputable and suggests an evolutionary connection. It is
 unknown whether GTAs were derived from pre-existing phage, whether the GTAs
 have a completely bacterial origin and gave rise to tailed phage, or if GTAs
 and phage evolved from undiscovered or extinct virus-like particle.

Direct cytoplasmic connection between neighboring cells is a common form of
communication in multicellular organisms, such as plasmodesmata in
plants\cite{Heinlein:2004fa} and gap junctions in mammals\cite{Kumar:1996tm}.
When growing upon a solid surface, bacterial cells can form networks of
tunneling nanotubes -- cytoplasmic connections between adjacent
cells\cite{Dubey:2011dp}. These connections have been shown to transport
cytoplasmic contents between cells including small-molecules, proteins, and even
plasmids. While nanotubes are short (up to 1\textmu m in length), they may play
an important role in HGT within biofilms.  This is of particular concern as they
are capable of passaging antibiotic resistance genes within single and
multi-species biofilms. It remains unknown how cargo is transported through the
nanotubes (active vs. passive diffusion), their molecular composition, or what
(if anything) regulates their formation.

It has been known for nearly 50 years that many species of gram-negative
bacteria readily produce outer membrane vesicles (MVs), although the function
and composition has only been the subject of inquiry for the past two decades.
Ranging from 50 - 250 nm is diameter and composed entirely of outer membrane,
MVs contain a diverse array of cargo such as toxins, signaling molecules,
various other proteins, and DNA (reviewed in \citet{MashburnWarren:2006jc}). MVs
may serve numerous biological roles ranging from predation and defense,
long-range communication, and as an avenue of HGT. What drove their evolution is
still not completely understood and their role in HGT may be an unintended
consequence. Regardless, their ability to transfer DNA not only between species
but between kingdoms garners further investigation into their role of genome evolution.



\subsection*{Evolutionary implications of HGT}

The ubiquity and evolutionary importance of HGT is becoming increasingly
apparent as more complete genomes are sequenced and made publicly available. As
the microbial world is not limited only to the set of genes present in its own
lineage, the "species tree" concept becomes difficult to interpret. From a
mathematical framework, trees are defined as a framework in which there are no
reticulations between nodes in a given branch. This type of model is not
applicable to the microbial world where genes can flow in multiple dimensions.

For nearly the past 60 years, the uniformity of the
genetic code amongst all life was described by one of two models. The
stereochemical model suggested a deterministic relationship between the chemical
properties of the codon and it's amino acid \cite{Grafstein:1983wv} whereas the
"frozen accident" model suggested that the genetic code could be arbitrary and
was universally adopted as all life evolved from the last universal common
ancestor\cite{Crick:1968wg}. There is a third possibility, however, that HGT
exerted a selective pressure to allow the exchange of genetic information
between species \cite{Syvanen:1985vv, Syvanen:2012jn}, forcing all life to adopt
and optimize the same (or at least very similar) genetic
code. \citet{Vestigian:2006va} demonstrated a unity between the stereochemical model
and HGT. They showed computationally that a pool of individuals connected by
avenues for HGT lead to the unification and optimization of a standard genetic
code. 

While HGT has received much experimental and theoretical attention over the past
century, there are still gaping holes in our understanding of the deeper
implications and influence on evolution as whole. Generating a complete picture
of how genetic information is passed between organisms (both through vertical
and horizontal) transmission is imperative not only to our understanding of the
origin and diversification of microbes, but of the entire biosphere.


HGT is particularly studied by looking for the transfer of operational or
protein-coding sequences. Horizontal transfer of genetic information is not
limited to genes.  Regulatory sequences can be exchanged\cite{Oren:2014eea}
leading to new and interesting patterns of regulation which can have an affect
on the composition of the environment that are subtle but important.

Horizontal transfer has the potential to drastically change the physiology of
the organism not only through the transfer of informational and operational
genes, but can rewire extant genetic circuits.

\section*{Experimental Approach} 
\subsection*{Watching DNA flow in real-time}
\begin{wrapfigure}{l}{0.5\textwidth}
	\centerline{\includegraphics[width=0.5\textwidth]{figs/par_function.eps}}
	\caption{Fluorescent labeling using the \textit{parABS} system. The {\bf
		\textit{parABS} segregation mechanism} under native conditions
		is described in the top panel. Upon binding of a ParB particle to a 
		\textit{parS} sequence, many other copies of parB are recruited
		and oligomerize on the plasmid. This complex is then recognized
	by ParA which polymerizes and forces the plasmids to opposite poles of
	the dividing cell. {\bf Plasmid labeling in \textit{E. coli}} is shown
	in the bottom panel. The image on the left shows \textit{E. coli} cells
	expressing \textit{YGFP-parB} system with a \textit{parS} containing
plasmid growing on a solid surface. The right image illustrates the source of
fluorescent puncta.}
	\label{fig:par_system}
\end{wrapfigure}



A vast majority of studies focusing on the frequency and efficiency of HGT in
microbes are performed in bulk -- often growing bacterial cultures to saturation
and measuring success of HGT by counting "colony forming units" (CFU). This leaves
many aspects of what happens at the cellular level unanswered, however, as CFU
serves as merely an approximation of cell number. In addition, microbial
populations are often far from saturation in natural
environments\cite{Vieira:2005jw, Whitman:1998tj} and thrive as structured,
non-homogenous mixtures known as biofilms \cite{HallStoodley:2004cv}. The
disparity in physiological relevance between the lab bench and the real-world
calls for single-cell and single-molecule techniques to study the dynamics and
evolutionary consequences of HGT.

A common approach to monitoring HGT is by selecting for some acquired gene, such
as an antibiotic resistance element or a fluorescent reporter. While these
methods are useful for bulk measurements, the time from acquisition to
observable phenotype is often longer than the typical bacterial division time as
the transferred element must be stabilized (not degraded), transcribed,
translated, and properly folded before they can combat the selection. While this
is particularly true for antibiotic resistance genes, the maturation time of
many fluorescent proteins range from ten minutes to two
hours\cite{Iizuka:2011ia}. It is therefore difficult to identify when the DNA
was transferred. Additionally, measuring HGT with selectable markers grossly underestimates the
frequency as one only measures the "winners" -- those events which avoided
degradation, were properly expressed, and did not kill the cell along the way.
It is likely that HGT occurs much more frequently but is missed by classical
measurements. To properly quantify the extent of HGT, one can not ignore the
"losers".


Although not focused on horizontal transfer, many molecular systems are in place
to  probe the sub-cellular localization of DNA elements using fluorescence
microscopy. Fluorescence In-Situ Hybridization (FISH) has been used extensively
over the past three decades to make such measurements\cite{Levsky:2003bz},
although the cell fixation required makes dynamical measurements impossible.
Over the past ten years, techniques for the sequence-specific labeling of DNA
\textit{In vivo} have been developed with varying degrees of applicability.
Repressor binding site arrays, such as \textit{lacO} and \textit{tetR}, have
been used to observe dynamic phenomena plasmid trafficking\cite{Ho:1911wf},
chromosomal rearrangements\cite{Lau:2004bp}, and telomere
mobility\cite{Jegou:2009kb} in living cells by fusing the repressor protein with
a fluorophore and observing localization. To be observable, however, there must
be many repeats of the binding sequence (often around 250) which not only
results in a large insertion of DNA (8 - 10 kbp) but can be physiologically
disruptive to the cell by interfering with DNA replication\cite{Dubarry:2011bx,
Payne:2006fc}. As HGT of such massive fragments is likely inefficient and the
physiological consequences are too severe, an alternative methodology is needed.


Only recently has such a system become available. Low-copy number autonomous
genetic elements, such as plasmids and chromosomes, often encode partitioning
systems which ensure faithful segregation into the daughter cells upon division.
Partitioning systems are diverse in strategy ranging from site-specific
recombination to post segregational killing mechanism to highly-specific and
highly-regulated DNA-protein complex formation. The latter is exceptionally
common and is often referred to as a Par system. This autoregulated mechanism of
segregation consists of three components (reviewed in \citet{Funnell:2004wi} and
\citet{Bignell:2001ti},
Fig. \ref{fig:par_system} top panel) -- a $\sim$ 100bp DNA binding site
(\textit{parS}), a DNA binding protein (ParB), and an ATPase (ParA). During the
first steps of partitioning, diffusive ParB molecules recognize and bind to an
available \textit{parS} sequence. Once bound, ParB experiences a conformational
change which promotes the cooperative association of other ParB dimers,
resulting in the recruitment of several hundred copies to the DNA. This complex
is very large and leads to non-specific binding of DNA outside of the
\textit{parS} sequence (a phenomenon known as "spreading"). In it's ATP bound
state, ParA non-specificially binds to the nucleoid in a cooperative manner
forming long polymers of ParA. Upon binding of the ParB-\textit{parS} complex to
ATP-bound ParA, hydrolysis occurs forcing depolymerization of the ParA-nucleoid
complex. ParB-bound plasmids can "chase" this receding wave to the poles of the
cell, resulting in faithful segregation.  This mechanism was only recently
observed although hypothesized through plasmid labeling via
\textit{tetO}-arrays\cite{Ringgaard:2009uh}.

By removing ParA from the system, the ParB-\textit{parS} complex remains
assembled but is doomed to drift aimlessly through cytoplasmic milieu. Attaching
fluorophores to the ParB results in the formation of fluorescent puncta (Fig.
\ref{fig:par_system} bottom panel). This system has been used to monitor
chromosomal rearrangements in numerous bacteria, but has never to my knowledge
been used to observe the transmission of mobile genetic
elements\cite{Nielsen:2006hmb, Nielsen:2007cta, Shebelut:2010ic,
Broedersz:2014jz}. Beginning with two \textit{parABS} homologs, four
fluorophores, and two organisms, this system has been successfully established
in \textit{E. coli} and \textit{Bacillus subtilis}, allowing for the
visualization and quantitation of HGT through transformation, transduction, and
conjugation in multiple species at single-cell resolution.


\subsection*{Measuring the limits and frequency of natural transformation}

\begin{wrapfigure}{r}{0.4\textwidth}
	\centerline{\includegraphics[width=0.4\textwidth]{figs/transformation_studies.eps}}
	\caption{Quantifying transformation in multiple species. Top panel shows
	experiments to test transformation frequency in bacillus based on DNA
parameters. Middle panel shows experiments to probe role of DNA uptake in
competence regulation. Bottom panel shows experiments to probe transformation in
E. coli.}
\label{fig:transformation_exp}
\end{wrapfigure}

Entry into the competence state is a requirement for uptake of free DNA from the
environment through natural transformation. Luckily, the regulatory networks
controlling entry and exit from the competence state has been studied
extensively in many model organisms such as \textit{Streptococcus
pyogoenes}\cite{Woodbury:2006dg}, \textit{Pseudomonas}\cite{Graupner:2001ii} and
\textit{Bacillus subtilis}\cite{Suel:2006ea,Suel:2007dm}, among others.  Those
mentioned are model organisms, affording us the opportunity to study the
dynamics of DNA uptake and persistence or degradation in developing colonies and
multi-species communities.

In collaboration with the Adam Rosenthal from the Michael Elowitz lab, I have
been able to translate this system into \textit{Bacillus subtilis} strains in
which entry into the competence state is under genetic control. We know that in cultures grown in
physiological conditions that only about 10\% of the cells are competent.  It
is unknown how many of those cells actually take up DNA and successfully
integrate the material into their genome. Using the ParBS system,  I am in the
unique position to make probe the extent at which cells enter competency and tak
eup DNA.  In \textit{Bacillus subtilis}, the
competence state can be turned on using common molecular biology tools. I will
be able to induce competence and watch for the flux of \textit{parS} DNA into
the cell by watching for the formation of puncta. 



Will all competent cells eventually take up DNA or is there some sweet spot?
Does duration of the competence state depend on DNA uptake? Do cells which took
up DNA have a shorter or longer competence state than cells who do not? Need to
do time lapse and record pComG activity. 

Does uptake of DNA signal exit from the competence state? One would imagine some
kind of receptor for uptake to avoid perpetually taking up DNA.	How does
frequency of uptake correlate to entry into the comptence state? 

How frequently is DNA taken up, but not successfully integrated? This can be
done by doing time lapse microscopy with forced entry into the competence state.

The presence of DNA has never been reported to stimulate competence in any
organism. As DNA is ubiquitous in the environment\cite{DellAnno:2002uw} and are
an integral component in biofilms\cite{Tang:2013kj}, this is not a surprising
result. However it is unknown whether the uptake of DNA may stimulate exit from
the competence state. 

Despite being possibly the worlds best-studied organism, the natural competence
of \textit{Eschericha coli} is still enigmatic. Several \textit{E. coli} genes
have been identified as homologs to competence-specific genes in other
gram-negative bacteria \cite{Averhoff:2003ex, Chen:2004iya}. A "natural
competence" state can be forced in \textit{E. coli} upon expression of a handful
of these homologs, allowing the use of DNA as a carbon
source\cite{Palchevskiy:2006kqb, Finkel:2001ge}. While expressing the competence
genes alone is sufficient to allow uptake of DNA, transformants never form,
suggesting the lack of appropriate DNA processing machinery to ensure
stabilization of the transferred fragment. Expressing the $\lambda$-red
recombinase along with the competence homologs readily produced
transformants\cite{Sinha:2012eha}. These studies suggest that \textit{E. coli}
posses the ability to acquire exogenous DNA through natural transformation,
although the processing machinery (if present) and the physiological signals
that stimulate expression of the competence genes is unknown. Using the
\textit{parBS} visualization system, coupled with what is learned from studying
transformation in \textit{Bacillus subilis}, I would like to examine the
frequency of DNA uptake in \textit{E. coli}. XXX



\subsection*{Observing transduction in phage P1 and $\lambda$}
Both conjugation and natural transformation rely on close proximity between the
DNA and accepting cell. To make matters worse, pilus attachment and the conjugal
bridges are highly unstable and subject to mechanical fracture. Free DNA in the
environment is highly susceptible to chemical decay meaning that mobilityTransduction is a
long-range form of gene transfer.  Viral elements are very stable in natural
environments (Find a citation for this).

Because of the persistence of viral particles, they have the opportunity to stay
in the environment through many generations of bacteria. This can include cycles
of succession and changes in the members of the population.  This means that
genes that were advantageous (or harmful) generations ago could come back into
play through transduction.

Like transformation, transduction has really only been studied in the bulk often
relying on a single infection of a phage, proper integration of the selectable
element, and then expression of the selectable element to generate a
transductant. 

The transduction efficiency varies from phage to phage. 

Phage P1 is a commonly used tool of molecular biology because it is very lax
with the DNA that it packages into its capsid. This raises the efficiency of
transduction to a measureable level. 

The experimental procedure involves integrating the \textit{parS} sequence into
several different loci in a host.  These could be surrounding different
metabolic genes, accessory genes,  core genes etc. By integrating many at the
same time, we increase the probability of having a detectable locus packaged
into the phage.

We can then infect cells that are expressing the YGFP-ParB construct and watch
for the formation of puncta. With a system where we can detect individual pieces
of DNA, we can ask all sorts of interesting questions. How viral particles can
transfer DNA at once? How frequently are there infections where the DNA is
degraded before integration occurs? How rapidly after infection are foreign
genes integrated into the genome versus degradation? 

\begin{figure}
	\centerline{\includegraphics[width=0.8\textwidth]{figs/transduction_experiments.eps}}
	\caption{Some caption describing experiment and testable questions.}
	\label{fig:transduction_exp}
\end{figure}

Previous studies in the Phillips lab have tracked the ejection of phage DNA
material into the cell. We can expand on that by tracking \textit{specific} bits
of genetic material.


\subsection*{Probing the positional and/or functional dependence of HGT}

Examining the transfer frequency of core and accessory genes is well within the
realm of feasibility in the experiments described above. However, this
positional/functional dependence of transferred genes can also be explored
through bioinformatics. 

Most methods of detecting horizontal gene transfer events focus on regions of
the chromosome which have either aberrant codon usage or aberrant GC content.
Most often, these studies are performed only on protein-coding regions leaving
non-coding and regulatory sequences behind. Recent work has shown that in E.
coli, the transfer of regulatory regions has happened extensively (Oren et al
2014) and could lead to the rewiring or formation of novel genetic circuits (not
from Oren, but my own general question). 

Previous bioinformatic studies have shown that operational rather than
informational genes are more prone to transfer [CITE]. This However this does
not take into consideration the neighboring genes that were transfered.

One could imagine that this may drive the evolution of the genome architecture.
Would there be a benefit to be selfish and put beneficial genes next to targets
that are likely to be incompatible with horizontal transfer?

Using the ever-growing database of microbial genomes, I would like to explore
the patterns of horizontally transferred genes and non-coding DNA. My prediction
is that genes laying near “core genes” (such as rRNA or RNAP genes) are less
likely to be transferred than if it were floating in a sea of non-coding
prophage remnants. One can imagine that a differential in the rate of horizontal
transfer can lead to the clustering of genes of similar function and/or
importance.  The big question I have  from this rambling is the following - does
horizontal gene transfer influence the architecture of the genome as a whole?

While I do not currently know how to do the bioinformatics, I know the tools are
out there and I’ll invest whatever time is necessary to learn them. 

All modern bioinformatic approaches to studying HGT look for genes or the
regions of the chromosome that have non-optimal codon usage, aberrant GC content
(compared to the rest of the genome), or novel functions that are not common to
close ancestral neighbors. No studies have looked at aberrant positioning of
genes when compared to ancestral neighbors. Can genes 'hitch-hike' on transfer
events of junk or non-coding DNA or vice versa? 

Did HGT play a role in the evolution of operons?


\section*{Concluding remarks}

Modern advancements in genome-editing, high-throughput and single-molecule
microscopy techniques, long-read sequencing, and computational approaches to
model living systems have placed us in the auspicious position to query the
molecular processes that drive all of evolution. A multidisciplinary approach
using tools from evolutionary biology and ecology, the mathematical logic from
physics, and modern biological techniques holds the potential to advance our
understanding of the biophysical details of biology's greatest idea --
evolution. This research will advance our understanding of how DNA migrates
between bacterial cells and contributes to evolution of their populations. This
work holds promise for a deeper understanding of the generation of multiple
antibiotic resistant bacteria. In addition, many important industrial processes
rely on the faithful propagation of large microbial communities (e.g. biofuel
production) or the growth of genetically modified organisms. Understanding how
genetic information flows within communities and between species is important
for development of novel industrial processes as well as for assessing the risk
posed by accidental release of genetically engineered organisms into the
environment. 


\section*{References}
\bibliographystyle{plainnat}
\bibliography{../../bib_files/library.bib}
\end{document}
