\input{../../tex_tools/candidacy_preamble.tex}

\title{Probing the Physical Basis of Genomic Promiscuity in Bacterial Evolution}
\author{Griffin Chure - Research Candidacy Proposal\\
	\textit{California Institute of Technology - Biochemistry and Molecular
	Biophysics}}
\date{Draft -- \today}

\begin{document}
\maketitle
\begin{abstract}
	Bacterial populations live in a fertile soup of foreign genetic
	material[CITATION]. Whether by viral infection, conjugative transfer, or
	direct uptake of DNA through transformation, bacteria frequently share
	their genetic information with their neighbors, including those of
	different species, strongly influencing bacterial evolution. In this
	research, I seek to characterize and quantify the frequency and dynamics
	of the horizontal transfer of genetic material at single-cell
	resolution. I have employed a genetically encoded system in \textit{E.
	coli} and \textit{Bacillus subtilis} that allows for the observation of
	specific DNA molecules as they are transfered between individual cells.
	The orthogonal nature of the visualization system allows for \textit{in
	vivo} studies. In addition, I wish to form a better understanding of the
	evolutionary consequences horizontal gene transfer on the assembly of
	microbial communities. 
\end{abstract}

\section*{Introduction}
Understanding the ebb and flow of genetic information through biological
lineages and communities has long been the focal point of biological inquiry.
Gregor Mendel set the stage for genetics. \cite{Syvanen:2012jn}

Recent advancements in sequencing and molecular biological techniques have shown
that vertical transmission of genetic material from parent to daughter cell is
not the whole story. 

\begin{itemize}
	\item Allows for an immediate response to changing environmental state.
	\item Cells that are more prone to horizontal gene transfer can reap the
		benefits from their dying neighbors. 
	\item The immediate change in geneotype 
\end{itemize}<++>
\begin{itemize}
	\item  
	\item Cite the recent paper proving that the marine snail shuttles
		photosynthesis genes from cyanobacteria into it's own genome and
		explain how this is so fucking amazing. Not only does the snail
		incorporate it, it expresses it!
	\item Explain that the genetic constructs built in the lab could get
		into the environment and permanently imprint themselves on the
		genome of nature. 
\end{itemize}
\subsection*{Conjugation}
Through the early 1940's, our understanding of bacterial phylogeny and
prokaryotic inheritance as a whole had been limited to the accumulation of
mutations through vertical transmission of genetic information from cell to
cell. While Frederick Griffith, Avery MacLeod, and others had shown that
bacteria could be "transformed" into pathogenicity by
DNA\cite{Griffith:1928vg}\cite{Avery:2014wx}, there had been little to no
investigation into its affect on bacterial evolution. The first evidence of
genetic recombination in bacterial specimens came from the work of Tatum and
Lederberg in 1947 which showed that bacteria can transfer information with their
neighbors by entering a "sexual stage."\cite{Tatum:1947va}

In the 70 years since the discovery, the molecular details of bacterial
conjugation have been studied extensively. Key features
\begin{itemize}
	\item Fertility element (HFR) encodes all genes necessary for transfer.
		This is also the genetic element that gets transfered. This
		means that so long as it successfully transfers in it entirety,
		the recieving cell will become able to pass along the plasmid. 
	\item The plasmid has a specific site \textit{oriT} where the transfer
		and begins and ends. It is at this site that relaxase proteins
		nick the plasmid allowing a single strand to be passed through
		the pilus.
	\item HFR-HFR transfers are rare or do not occur at all (NEED CITATION).
		This indicates that there is some molecular marker which HFR
		cells can identify viable targets.
	\item Primary mechanism for horizontal gene transfer. This is very
		short-range, however, as the cells need to be close enough to
		touch each other. This means that conjugal transfer in biofilms
		is expected and incredibly common. 
	\item Because a F element is needed with a specific start and end site,
		cells are limited in what genes can be transfered. Unlike in
		transformation and transduction, it is very specific in it's transfer. 
	
\end{itemize}

The features of the pilus are as follows.
\begin{itemize}
	\item The transmembrane conduit is related to the typeIV secretion
		system. It is composed of about ten different proteins that span
		both the inner and outer membrane of gram-negative bacteria.
	\item Need more details. Talk about the Ecoli F pilus and F element and
		how it contrasts the TypeIV pilus.
\end{itemize}


Many of the experiments which characterized this system were done \textit{in
vitro}. This leaves many of the physiological questions unanswered. It is still
unknown what events occur in the recipient cell to stabilize the plasmid as it
is transfered as a single strand. It is also unknown how the transfer process
interfaces with the myriad of other DNA maintenance processes such as
replication and partitioning. 

There have been some \textit{in vivo} which have helped shed light on how the
conjugal elements traverse bacterial communities. Briefly (in five or six
sentences) summarize the babic paper. \cite{Babic:2008bl}

\begin{itemize}

	\item Brief overview of how it was discovered and how it led to the
		minute definition of the bacterial genome.
	\item Requires the presence of a fertility element.
	\item Requires the assembly of a large pilus. Depends on other bacteria
		being very close together
	\item Keep this section short
\end{itemize}

\subsection*{Transduction}
\subsection*{Transformation}
\begin{itemize}
	\item Natural transformation allows for gene transfer between organisms
		who are not in the same host range for bacteriophage
		(transduction) or conjugation.
	\item 
\end{itemize}<++>
\subsection*{Special Cases}
\begin{itemize}
	\item Nanotubes.
	\item Gene Transfer Agents.
\end{itemize}
\subsection*{Evolutionary implications of HGT}
\begin{itemize}
	\item 	
	\item Destruction of the 'tree' model of life. A network is more
		appropriate, although computationally intractable.
	\item Promoted the use of the universal genetic code? There is an
		immense selective pressure of translating a library than
		learning the language. 
	\item Exacerbates the "species problem" of microbes. I should give the
		example that an alignment of all of the 70 \textit{E. coli}
		genomes reveals only something like 700 genes in common. Why is
		there so much variation in the genome? If the genomes of the
		entire environment is available how can you define differences?
	\item Generating a complete picture of how genes are passed around (and
		how frequently they do it) is imperative to the understanding of
		the evolution of not only microbes, but of the entire biosphere. 
	\item Horizontal transfer of genetic information is not limited to
		genes. Regulatory sequences can be exchanged (cite the HRT
		paper) leading to new and interesting patterns of regulation
		which can have an affect on the composition of the environment
		that are subtle but important.
	\item I should estimate what portion of sequence space has been explored
		in all of life. I should then estimate how long it would take a
		single cell line to explore the same space. This should show
		that HGT speeds up the process immensely, allowing cells to
		speed-read sequence space without having to do all  of the heavy
		lifting.
\end{itemize}
\section*{Experimental Approach}
\subsection*{Watching DNA flow in real-time}
\begin{itemize}
	\item With all of the evidence of HGT, it's often impossible to
		understand how it moved around relying on sequence alone. 
	\item Bulk-scale studies are almost certainly and underestimate of the
		frequency. Fluctuations at the scale that are relevant to
		environmental communities  (10$^2$ - 10$^3$ cells) [I need to
		find a citation for this] are drowned out when doing the
		experiments with saturated cultures (10$^9$ cells). Getting a
		picture at the single-cell level will greatly advance our
		understanding of how frequently genes are passed around and will
		let us quantify the effect it has on the development of
		communities. The effects felt by HGT begin with a single cell.
		Being able to watch the first several cell divisions will help
		us measure these effects.
	\item Classical fluorescent reporters are useful but rely on a large
		series of complex events to be detected. They must be properly
		integrate, transcribed, translated, and then properly mature
		before they are detectable. This puts a limit on the temporal
		resolution although gives a readout of Whether the genes are
		expressed or not. It would be great to watch the DNA travel
		inside of cells and between cell in an orthogonal manner would
		be fucking awesome and abolish the problems of traditional reporters. 
	\item Labs in the past [CITE some of Pogliano's work] rely on large
		arrays of repressor binding sites. These often result in immense
		portions of the DNA being dedicated to the visualization. This
		is a large perturbation. To really understand the dynamics of
		horizontal gene transfer, you need a system that perturbs the
		DNA in the most minimal manner. Ideally, this would be something
		that is genetically tractable and orthogonal to the cell. This
		seems like it's asking a lot, but we already have that. 
	\item ParABS system is used by low copy-number plasmids to ensure
		faithful and equal inheritance in dividing cells. This is
		composed of three partners. ParA is the atpase which drives the
		active segregation, ParB is the protein which binds to the short
		($\sim$100 bp) \textit{parS} DNA sequence. This binding is
		cooperative. Once ParB is bound to the DNA, the conformation
		changes prompting cooperative association of other ParB
		proteins. This results in an effect called 'spreading' where the
		ParB proteins bind to the DNA nonspcifically but tightly because
		of the cooperative nature of ParB. This results in a high
		concentration of the ParB proteins in a very small volume. By
		fluorescently tagging ParB, removing ParA, and cloning
		\textit{parS} into regions of interest, we can fluorescently tag
		DNA specifically and orthogonally with only minor sequence
		perturbations. This system serves as an "instantaneous
		reporter" that does not rely on any of the issues described
		above with canonical reporter systems.
	\item Explain and use a figure to show that in the absence of DNA, the
		cell is uniformly bright. When \textit{parS} is present, a
		bright focus appears. 

	\item I need make it clear what I have done on this system so far. I
		should explain how I tried to make the two color system work but
		decided a one-color system will allow for more simple
		measurements. I can describe how I've integrated into different
		positions and have tried to optimize expression to allow for
		detection of low copy number elements. Make it clear that I've
		tried using several different fluorophores, promoters, RBSs, etc
		to get this working. This system has been used to observe the
		spatial organization of the chromosome, but has never been
		used to watch the flux of gene elements through the cell. This
		has required more work than initially thought. 
\end{itemize}
\subsection*{Measuring frequency of natural transformation in genetically
tractable systems}
\begin{itemize}
	\item The ParBS system allows for measurement of gene transfer without
		having to rely on the expression of the transferred genetic material.
	\item I've been trying to observe the uptake of plasmid DNA during the
		run-of-the-mill molecular biology techniques of electroporation
		and transformation, but have run into issues. 
	\item I should show a figure the results of electroporation and
		transformation and how the formation of aggregates and inclusion
		bodies has confounded the situation. 
	\item List the strategies I have tried and am currently trying to
		develop to watch DNA transfer into cells. I need to explain what
		we will learn from this type of measurement (focusing again on
		the inaccuracy of bulk measurement methods). I can emphasize
		that chemical transformation happens in nature (cite the E Coli
		natural transformation paper). And that being able to understand
		the frequency of transfer is important physiologically and
		environmentally. 
	\item Due to the problems with \textit{E. coli} and its enigmatic
		competence state, I decided to switch gears to an organism who's
		competence state is more understood and is genetically
		tractable. 
	\item In collaboration with the Elowitz lab, I have (or will be at
		least) translated this system into \textit{Bacillus subtilis},
		who's competence state is well understood and is genetically
		tractable. 
	\item We know that in cultures grown in physiological conditions that
		only about 10\% of the cells are competent.  It is unknown how
		many of those cells actually take up DNA and successfully
		integrate the material into their genome. Using the ParBS
		system,  I am in the unique position to make this measurement.
		In \textit{Bacillus subtilis}, the competence state can be
		turned on using common molecular biology tools. I will be able
		to induce competence and watch for the flux of \textit{parS} DNA
		into the cell by watching for the formation of puncta. 
	\item I plan to also translate this system into other organisms whose
		competence states are more understood (such as
		\textit{Streptococcus pyogenes} and \textit{pneumoniae}) and see
		what kind of differences there are between the frequency of
		uptake and the frequency of entry to competence. I would then be
		interesting to study how DNA is transferred between species.
	\item Using what I have learned from the well-characterized systems, I
		can take this back to \textit{E. coli} and resume these studies.
\end{itemize}
\subsection*{Observing transduction in phage P1 and $\lambda$}
	\begin{itemize}
		\item Both conjugation and natural transformation rely on being
			within close proximity of the DNA. DNA doesn't last very
			long in natural environments and conjugation relies on
			bacteria being very close to each other spatially.
			Transduction is a long-range form of gene transfer.
			Viral elements are very stable in natural environments
			(Find a citation for this).
		\item Because of the persistence of viral particles, they have
			the opportunity to stay in the environment through many
			generations of bacteria. This can include cycles of
			succession and changes in the members of the population. 
			This means that genes that were advantageous (or harmful
			) generations ago could come back into play through
			transduction.
		\item The transduction efficiency varies from phage to phage. 
		\item I should discuss the different ways of packaging host
			genome (headful vs. any DNA vs. other mechanisms).
			Phage genomes are often several 10's of kilobases.
			This means that many genes, even entire operons and
			metabolic circuits, could be transferred horizontally
			between organisms from a single transduction event. 
		\item Like transformation, transduction has really only been
			studied in the bulk often relying on a single infection
			of a phage, proper integration of the selectable
			element, and then expression of the selectable element
			to generate a transductant. 
		\item We would learn much from observing transduction at the
			single cell level. We could get a better handle on the
			frequency of transduction as well as how many trasducing
			particles can infect a cell at once. Using the ParBS
			system, we could be able to watch the DNA as it is
			integrated into the genome by measuring the difference
			in diffusion of the puncta.
		\item Transduction often occurs at a low frequency, however,
			meaning we either need some way to scan through many,
			many cells and infections at once or we need to boost up
			the frequency of transduction.
		\item Phage P1 is a commonly used tool of molecular biology
			because it is very lax with the DNA that it packages
			into its capsid. This raises the efficiency of
			transduction to a measureable level. 
		\item The experimental procedure involves integrating the
			\textit{parS} sequence into several different loci in a
			host. These could be surrounding different metabolic
			genes, accessory genes,  core genes etc. By integrating
			many at the same time, we increase the probability of
			having a detectable locus packaged into the phage.
		\item We can then infect cells that are expressing the YGFP-ParB
			construct and watch for the formation of puncta. With a
			system where we can detect individual pieces of DNA, we
			can ask all sorts of interesting questions. How viral
			particles can transfer DNA at once? How frequently are
			there infections where the DNA is degraded before
			integration occurs? How rapidly after infection are
			foreign genes integrated into the genome? How does this
			integration frequency compare with the phage genome?
		\item Previous studies in the Phillips lab have tracked the
			ejection of phage DNA material into the cell. We can
			expand on that by tracking \textit{specific} bits of
			genetic material.
	\end{itemize}
\subsection*{Observing the influence of HGT on community organization}


\section*{Concluding remarks}
All of evolution is driven by molecular behavior within the cell. While there has
been much advancement in both evolutionary and cellular biology, there has been
remarkably little overlap between them. A multidisciplinary approach using tools
from evolutionary biology and ecology, the mathematical logic from physics, and
modern biological techniques holds the potential to advance our understanding of
the biophysical details of biology’s greatest idea
– evolution. This research will advance our understanding of how DNA migrates
between bacterial cells and contributes to evolution of their populations. This
work holds promise for a deeper understanding of the generation of multiple
antibiotic resistant bacteria. In addition, many important industrial processes
rely on the faithful propagation of large micro- bial communities (e.g. biofuel
production) or the growth of genetically modified organisms. Understanding how
genetic information flows within communities and between species is important
for development of novel industrial processes as well as for assessing the risk
posed by accidental release of genetically engineered organisms into the
environment. Blah blah other stuff


\section*{References}
\bibliographystyle{abbrv}
\bibliography{infield}{}
\end{document}
