\input{../../tex_tools/candidacy_preamble.tex}

\title{Probing the Physical Basis of Genomic Promiscuity in Bacterial Evolution}
\author{Griffin Chure - Research Candidacy Proposal\\
	\textit{California Institute of Technology - Biochemistry and Molecular
	Biophysics}}
\date{Draft -- \today}

\begin{document}
\maketitle
\begin{abstract}
	Bacterial populations live in a fertile soup of foreign genetic
	material[CITATION]. Whether by viral infection, conjugative transfer, or
	direct uptake of DNA through transformation, bacteria frequently share
	their genetic information with their neighbors, including those of
	different species, strongly influencing bacterial evolution. In this
	research, I seek to characterize and quantify the frequency and dynamics
	of the horizontal transfer of genetic material at single-cell
	resolution. In addition, I wish to form a better understanding of the
	evolutionary consequences horizontal gene transfer on the assembly of
	microbial communities. XXX
\end{abstract}

\section*{Introduction}
\begin{itemize}
	\item Cite the recent paper proving that the marine snail shuttles
		photosynthesis genes from cyanobacteria into it's own genome and
		explain how this is so fucking amazing. Not only does the snail
		incorporate it, it expresses it!
	\item Explain that the genetic constructs built in the lab could get
		into the environment and permanently imprint themeselves on the
		genome of nature. 
\end{itemize}
\subsection*{Conjugation}
\subsection*{Transduction}
\subsection*{Transformation}
\subsection*{Evolutionary implications of HGT}
\begin{itemize}
	\item Destruction of the 'tree' model of life. A network is more
		appropriate, although computationally intractable.
	\item Promoted the use of the universal genetic code? There is an
		immense selective pressure of translating a library than
		learning the language. 
	\item Exacerbates the "species problem" of microbes. I should give the
		example that an alignment of all of the 70 \textit{E. coli}
		genomes reveals only something like 700 genes in common. Why is
		there so much variation in the genome? If the genomes of the
		entire environment is available how can you define differences?
	\item Generating a complete picture of how genes are passed around (and
		how frequently they do it) is imperative to the understanding of
		the evolution of not only microbes, but of the entire biosphere. 
	\item Horizontal transfer of genetic information is not limited to
		genes. Regulatory sequences can be exchanged (cite the HRT
		paper) leading to new and interesting patterns of regulation
		which can have an affect on the composition of the environment
		that are subtle but important.
	\item I should estimate what portion of sequence space has been explored
		in all of life. I should then estimate how long it would take a
		single cell line to explore the same space. This should show
		that HGT speeds up the process immensely, allowing cells to
		speed-read sequence space without having to do all  of the heavy
		lifting.
\end{itemize}
\section*{Experimental Approach}
\subsection*{Watching DNA flow in real-time}
\begin{itemize}
	\item With all of the evidence of HGT, it's often impossible to
		understand how it moved around relying on sequence alone. 
	\item Bulk-scale studies are almost certainly and underestimate of the
		frequency. Fluctuations at the scale that are relevant to
		environmental communities  (10$^2$ - 10$^3$ cells) [I need to
		find a citation for this] are drowned out when doing the
		experiments with saturated cultures (10$^9$ cells). Getting a
		picture at the single-cell level will greatly advance our
		understanding of how frequently genes are passed around and will
		let us quantify the effect it has on the development of
		communities. The effects felt by HGT begin with a single cell.
		Being able to watch the first several cell divisions will help
		us measure these effects.
	\item Classical fluorescent reporters are useful but rely on a large
		series of complex events to be detected. They must be properly
		integrate, transcribed, translated, and then properly mature
		before they are detectable. This puts a limit on the temporal
		resolution although gives a readout of Whether the genes are
		expressed or not. It would be great to watch the DNA travel
		inside of cells and between cell in an orthogonal manner would
		be fucking awesome and abolish the problems of traditional reporters. 
	\item Labs in the past [CITE some of Pogliano's work] rely on large
		arrays of repressor binding sites. These often result in immense
		portions of the DNA being dedicated to the visualization. This
		is a large perturbation. To really understand the dynamics of
		horizontal gene transfer, you need a system that perturbs the
		DNA in the most minimal manner. Ideally, this would be something
		that is genetically tractable and orthogonal to the cell. This
		seems like it's asking a lot, but we already have that. 
	\item ParABS system is used by low copy-number plasmids to ensure
		faithful and equal inheritance in dividing cells. This is
		composed of three partners. ParA is the atpase which drives the
		active segregation, ParB is the protein which binds to the short
		($\sim$100 bp) \textit{parS} DNA sequence. This binding is
		cooperative. Once ParB is bound to the DNA, the conformation
		changes prompting cooperative association of other ParB
		proteins. This results in an effect called 'spreading' where the
		ParB proteins bind to the DNA nonspcifically but tightly because
		of the cooperative nature of ParB. This results in a high
		concentration of the ParB proteins in a very small volume. By
		fluorescently tagging ParB, removing ParA, and cloning
		\textit{parS} into regions of interest, we can fluorescently tag
		DNA specifically and orthogonally with only minor sequence
		perturbations. This system serves as an "instantaneous
		reporter" that does not rely on any of the issues described
		above with canonical reporter systems.
	\item Explain and use a figure to show that in the absence of DNA, the
		cell is uniformly bright. When \textit{parS} is present, a
		bright focus appears. 
\end{itemize}
\subsection*{Measuring frequency of natural transformation in genetically
tractable systems}
\subsection*{Observing transduction in phage P1 and $\lambda$}
\subsection*{Quantification of intra-species gene transfer and its influence on
community organization}


\section*{Concluding remarks}

\end{document}
